docs/source/getting-started/install.rst

```diff
@@ -1,30 +1,32 @@
 Comprehensive Installation
 ==========================
 
-The fastest way to install Scancode-Toolkit is by using ``pip``. You can also install
-ScanCode-Toolkit by compiling it from source or by Downloading and Configuring the
-latest release from GitHub.
+There are 3 main ways you can install ScanCode.
 
-- :ref:`pip_install`
-- :ref:`latest_release_download_install`
-- :ref:`source_configure_install`
+- :ref:`app_install`
 
-.. NOTE::
+    The recommended method for installing ScanCode is Downloading the latest release, as an
+    application and then configure and use directly. This is easy because no knowledge of pip/git
+    or other developer tools is necessary.
 
-    After ``pip install``, you can perform a scan using only::
+- :ref:`pip_install`
 
-        scancode [OPTIONS] <OUTPUT FORMAT OPTION(s)> <SCAN INPUT>
+    The fastest way to install ScanCode as a python package, using ``pip``. You can also install
```

Comment:
Instead of saying "The fastest way", I think this should be "To use ScanCode as a library in your application, you can install it via pip. 

For the most part, we want beginners and most users to use the latest Github Release.